scripts/!mods_preload/msu.nut

```diff
@@ -1,30 +1,38 @@
-::MSU.EndQueue <- {
-	Queue = [],
+::MSU.HooksMod <- ::Hooks.register(::MSU.ID, ::MSU.Version, ::MSU.Name);
+::MSU.HooksMod.require(::MSU.VanillaID + " >= 1.5.0-13");
```

Comment:
We should remove this and all other references to `VanillaID` because that's now defined by modern hooks, we should instead probably just use `"vanilla"` since that's hardcoded by modern hooks